test/Api.ts/Tracks.test.js

```diff
@@ -1,5 +1,4 @@
-
-
+/* eslint-disable no-unused-expressions */
```

Comment:
@mattburnett-repo 

check out this eslint-disable rule here to make it so that `standard` won't complain about:

```
    expect(theData.cover_metadata.id).to.be.null
```

(`.null` should normally be a function call under normal javascript, which mocha seems to ignore)

Reply:
cool. I will add this to the test files. thanks